client/scss/components/forms/_title.scss

```diff
@@ -0,0 +1,32 @@
+// Styles for title fields throughout the page.
+.title {
```

Comment:
This classname may already exist - but this feels overly generic and likely to clash with other styles. - can we scope these styles any better?

Reply:
Indeed! I attempted to preserve this changing of the font styles because I wasn’t sure where exactly people were meant to reuse this, but really it should just be at the field-level or panel-level customisation. So I’ve scoped this further now.